apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V58__Add_rif_records_skipped_Table.sql

```diff
@@ -0,0 +1,40 @@
+/**
+ * Adds a new table for storing CCW RIF records that have been temporarily skipped.
+ *
+ * Please note that every record in this table is a piece of as-yet-unaddressed technical debt, with an
+ * exorbitantly high "interest rate": the older these records get, the harder it will be to figure out what
+ * to do with them, and the harder it will be to implement any such process. Ideally, this table is only ever
+ * populated for VERY brief periods of time.
+ * 
+ * Added as part of <https://jira.cms.gov/browse/BFD-1566>.
+ */
+
+CREATE TABLE skipped_rif_records (
+  -- Sequence-generated PK for these records, as no natural PK is feasible.
+  record_id            bigint                     NOT NULL PRIMARY KEY,
+
+  -- The timestamp associated with the CCW data set manifest that this record is from.
+  rif_file_timestamp   timestamp with time zone   NOT NULL,
+
+  -- The `RifFileType` (e.g. `BENEFICIARY`, `CARRIER`, etc.) of the RIF file that this record is from.
+  rif_file_type        varchar(48)                NOT NULL,
+
+  -- The `DML_IND` of the RIF record(s).
+  dml_ind              varchar(6)                 NOT NULL,
+
+  -- The `bene_id` of the beneficiary that this record is of / associated with.
+  bene_id              bigint                     NOT NULL,
```

Comment:
Logan, not sure how you guys worked the bene_id out, but the current db treats it as a varchar(15). Tratings it as a bigint is what the schema changes are heading towards, but we aren't there yet. Leaving as a varchar is not wrong by any stretch, and we can always cast as needed when reconciling this table with future claims processing.

Reply:
the thinking was that since this table doesn't really need an FK (nice to have one day though) that we save a future migration effort and make it bigint now and follow it up with an FK only migration once we have the beneficiaries table switched over.

Reply:
It depends on how we plan to use it:
1. If we aren't going to use it at all, for now, then having it be a `bigint` likely makes the most sense, as that's where the rest of the schema is headed and we avoid future conversion work.
2. If we're going to query the table separately, same answer as #1: keep it as a `bigint`.
3. If we're going to query it in `JOIN`s, might make more sense to make it a `varchar` for now, to keep the joins simpler.

Thoughts?

Reply:
Trotter's point re: FKs I think it more important than the query strategy, and I agree with him: no FK for now.

Reply:
The other possibility is to also add a 'temporary' column for bene_id varchar (i.e., bene_id_string) with an index. When we get to the full bigint support in the schema, we simply drop the added column.